files/templates/alertmanager/alertmanager.yaml

```diff
@@ -127,7 +127,11 @@ receivers:
 
 - name: team_atlas_slack
   slack_configs:
+  [[- if eq .Pipeline "stable" ]]
```

Comment:
Maybe we can keep giant swarm stable installations in #alert-atlas-test ?

Reply:
Too much noise imo

Reply:
Yes let's keep stable in #alert-atlas and testing in #alert-atlas-test